Meta/Lagom/CMakeLists.txt

```diff
@@ -200,6 +191,17 @@ if (BUILD_LAGOM)
                 WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
             )
         endforeach()
+
+        foreach(source ${LIBCOMPRESS_TESTS})
+            get_filename_component(name ${source} NAME_WE)
+            add_executable(${name}_lagom ${source} ${LIBCOMPRESS_SOURCES})
+            target_link_libraries(${name}_lagom Lagom)
```

Comment:
The AK and LibRegex tests above only link with LagomCore, would that be sufficient here as well?

Reply:
We need full Lagom here because some LibCompress functions depend on LibCrypto. 

If we try to link only using LagomCore: 
```
/usr/bin/ld: serenity/Build/i686/../../Userland/Libraries/LibCompress/Gzip.cpp:241: undefined reference to `Crypto::Checksum::CRC32::digest()'
```

Reply:
Thanks for clarifying!